test/cli.mjs

```diff
@@ -95,7 +112,6 @@ describe("postject CLI", () => {
           console.log(err.message);
         }
         if (codesignFound) {
-          execSync(`codesign --sign - ${filename}`);
```

Comment:
For some reason, code signing the binary after running `objcopy --add-section` on macOS causes the binary to crash:
```sh
$ ./node
[1]    5366 killed     ./node
$ echo $?
137
```

and I can't even `lldb` into it:

```sh
lldb -- ./node
(lldb) target create "./node"
Current executable set to '/Users/raisinten/Desktop/git/postject/node' (x86_64).
(lldb) run
error: Cannot allocate memory
(lldb) ^D
```

However, it seems to be possible to run `objcopy` after code signing and not invalidate the code signature! I also tried to check if the new section was being added after the `LC_CODE_SIGNATURE` section that @bnoordhuis mentioned in https://github.com/nodejs/node/pull/45066#issuecomment-1287431258 and it seems like `objcopy` adds new sections at the end, even after `LC_CODE_SIGNATURE` but the `codesign` tool thinks that this is valid:

```txt
$ otool -l ./node | tail -n29
Load command 18
      cmd LC_CODE_SIGNATURE
  cmdsize 16
  dataoff 84875008
 datasize 663232
Load command 19
      cmd LC_SEGMENT_64
  cmdsize 152
  segname __POSTJECT
   vmaddr 0x00000001057d4000
   vmsize 0x0000000000001000
  fileoff 68509696
 filesize 4096
  maxprot 0x00000007
 initprot 0x00000007
   nsects 1
    flags 0x0
Section
  sectname __NODE_JS_CODE
   segname __POSTJECT
      addr 0x00000001057d4000
      size 0x0000000000000014
    offset 68509696
     align 2^0 (1)
    reloff 0
    nreloc 0
     flags 0x00000000
 reserved1 0
 reserved2 0
```

```sh
$ codesign -v ./node
$ echo $?
0
```

Any clue what's happening here?